mlflow/tracing/display/display_handler.py

```diff
@@ -41,6 +41,10 @@ def enable(cls):
         if cls._instance is None:
             cls._instance = IPythonTraceDisplayHandler()
 
+    @property
+    def disabled(self):
+        return self._disabled
+
```

Comment:
@B-Step62 is this how it's usually done?

i originally had this as:
```python
@classmethod
@property
    def disabled(self):
        return self._disabled
```

but i get a warning telling me that doing this is deprecated (`Class properties are deprecated in Python 3.11 and will not be supported in Python 3.13`). i wonder if `_disabled` should just be an instance attribute instead

Reply:
Oh my bad yea it is class level thing. Yea I think it's ok to use attribute `disabled` (public).

Reply:
ah okay, so i'll just change the class attribute from `_disabled` -> `disabled`